plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -63,7 +63,7 @@ def _get_repo_unit_nevra(self, erratum_unit):
         nevra_in_repo_and_pkglist = []
         for pkglist in erratum_unit.pkglist:
             for pkg in pkglist['packages']:
-                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'],
+                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'] or '0',
```

Comment:
@dkliban ,
It is not possible if an erratum was [imported via sync](https://github.com/pulp/pulp_rpm/blob/pulp-rpm-2.9.2-1/plugins/pulp_rpm/plugins/importers/yum/repomd/updateinfo.py#L116).
But you can upload an erratum which will contain nearly anything in its pkglist, there is almost no validation for the erratum upload. I can file an issue if there is not any, but I suggest not to validate only epoch and and not to perform validation here, in publish step.
What do you think?